File: GO/ex1011.go
package main

import (
	"fmt"
)

func sphere(R float64, pi float64) float64{
	var volume float64
	volume = ((4/3.0) * R * R * R * pi)
	return volume
}

func main1011(){
	var pi float64
	var R float64
	pi = 3.14159
	fmt.Scanf("%f", &R)
	fmt.Printf("VOLUME = %.3f\n", sphere(R, pi))
}

File: C/ex1011.c
#include <stdio.h>

double sphere(double R, double pi);

int main()
{
    double     pi = 3.14159;
    double  R;
    
    scanf("%lf", &R);
    printf("VOLUME = %.3lf\n", sphere(R, pi));
    return (0);
}

double  sphere(double R, double pi)
{
    return ((4/3.0)*R*R*R*pi);
}